Makefile

```diff
@@ -70,7 +70,7 @@ test-merge:
 	  FILES=$$(git diff --name-only FETCH_HEAD `git merge-base FETCH_HEAD master 2>/dev/null` | grep 'tests.*pm') ;\
 	  for file in $$FILES; do if test -f $$file; then \
 	    tools/check_metadata $$file || touch failed; \
-	    git grep -q wait_idle $$file && touch failed; \
+	    git --no-pager grep wait_idle $$file ||  touch failed; \
```

Comment:
why are you turning around the logic? Grep succeeds when it *finds* a match.

Reply:
I think it's a typo. Sorry.